src/main/java/lotto/domain/Prize.java

```diff
@@ -0,0 +1,44 @@
+package lotto.domain;
+
+import java.util.*;
+
+public enum Prize {
+
+    FIRST(6, 2000000000),
+    SECOND(5, 1500000),
+    THIRD(4, 50000),
+    FOURTH(3, 5000),
+    ETC(0, 0);
```

Comment:
```suggestion
    FIRST(6, 2_000_000_000),
    SECOND(5, 1_500_000),
    THIRD(4, 50_000),
    FOURTH(3, 5_000),
    ETC(0, 0);
```
이렇게 언더바(`_`)를 이용하여 숫자를 표기하면 더 가독성이 좋습니다!

Reply:
이렇게 표기할 수 있는지 처음 알았네요! 참고하겠습니다.